### DISCUSSION TITLE
Matek f405 STD

### DISCUSSION THREAD

**Discussion by rotorrad** [iNavFlight/inav#6654] [Q&A]
Hi folks.
I'm new to small boards like omnibus, Matek, etc. I wanted to build a small fun quadcopter for the first time. I normally use Naza or eagle tree for 450+ quads. I took my time, started with omnibus F4 V4 had a problem with every radio that I own, probably a bad clone. Got the quad built with several revisions, not much space for what I'm used to.
Anyways finished built with Matek 405 STD, Matek VTXPDB hub, and Matek Ublox mn8. All in blue on 13 sats outdoors able to arm, etc. Radiomaster tx set up with three pos flight modes + arming switch. Modes are Angle, Pos Hold, Failsafe/RTH
Arms fine outside, but won't let me arm indoors in Angle mode, angle mode doesn't need GPS fix right?
![IMG_20210210_194645](https://user-images.githubusercontent.com/41281283/109485630-cfbf8a00-7a79-11eb-9589-05309f7c6675.jpg)

When I was testing it outside (luckily without props) after switching to failsafe, motor 3 shuts down completely.
What have I done wrong?

**Comment by rts18** [comment]
The Angle arming problem could be settings related. Not sure about motor 3 shutting down.  Are all the ESC's configured the same?
Please paste your diff all settings.

**Reply by rotorrad** [reply]
Hi thank you for your reply
My settings are as follow:

# version
# INAV/MATEKF405 2.6.0 Dec  4 2020 / 11:55:31 (cada68e4)
# GCC-9.2.1 20191025 (release) [ARM/arm-9-branch revision 277599]

# start the command batch
batch start

# reset configuration to default settings
defaults noreboot

# resources

# mixer

mmix reset

mmix 0  1.000 -1.000  1.000 -1.000
mmix 1  1.000 -1.000 -1.000  1.000
mmix 2  1.000  1.000  1.000  1.000
mmix 3  1.000  1.000 -1.000 -1.000

# servo mix

# servo

# safehome

# logic

# gvar

# feature
feature GPS
feature LED_STRIP
feature PWM_OUTPUT_ENABLE

# beeper

# map

# serial
serial 3 2 115200 115200 0 115200

# led
led 0 5,13:NS:CTW:2
led 1 6,13:NS:CTW:2
led 2 7,13:NS:CTW:2
led 3 8,13:NS:CTW:2
led 4 9,13:NS:CTW:2
led 5 10,13:NS:CTW:2
led 6 5,14:NS:CTW:2
led 7 6,14:NS:CTW:2
led 8 7,14:NS:CTW:2
led 9 8,14:NS:CTW:2
led 10 9,14:NS:CTW:2
led 11 10,14:NS:CTW:2
led 12 5,15:NS:CTW:2
led 13 6,15:NS:CTW:2
led 14 7,15:NS:CTW:2
led 15 8,15:NS:CTW:2
led 16 9,15:NS:CTW:2
led 17 10,15:NS:CTW:2

# color

# mode_color

# aux
aux 0 0 0 1800 2100
aux 1 1 2 900 1200
aux 2 11 2 1300 1700
aux 3 27 2 1800 2100

# adjrange

# rxrange

# temp_sensor

# wp
#wp 0 invalid

# osd_layout
osd_layout 0 0 25 2 V
osd_layout 0 1 14 2 V
osd_layout 0 9 1 2 H
osd_layout 0 10 1 7 H
osd_layout 0 11 2 2 V
osd_layout 0 12 1 3 V
osd_layout 0 14 2 13 V
osd_layout 0 15 25 3 V
osd_layout 0 25 23 5 V
osd_layout 0 28 24 13 V
osd_layout 0 48 23 7 V

# master
set looptime = 500
set gyro_hardware_lpf = 256HZ
set gyro_lpf_hz = 110
set gyro_lpf_type = PT1
set dynamic_gyro_notch_enabled = ON
set dynamic_gyro_notch_q = 250
set dynamic_gyro_notch_min_hz = 120
set acc_hardware = MPU6500
set acczero_x = -10
set acczero_y = -59
set acczero_z = -66
set accgain_x = 4073
set accgain_y = 4205
set accgain_z = 4056
set mag_hardware = QMC5883
set magzero_x = 353
set magzero_y = -103
set magzero_z = -100
set maggain_x = 1594
set maggain_y = 1725
set maggain_z = 1571
set baro_hardware = BMP280
set pitot_hardware = NONE
set blackbox_rate_denom = 2
set motor_pwm_rate = 4000
set motor_pwm_protocol = DSHOT150
set failsafe_procedure = RTH
set model_preview_type = 3
set applied_defaults = 2
set gps_sbas_mode = AUTO
set gps_ublox_use_galileo = ON
set mc_airmode_type = THROTTLE_THRESHOLD
set i2c_speed = 800KHZ
set name = Little Bee
set tz_automatic_dst = EU

# profile
profile 1

set mc_p_pitch = 44
set mc_i_pitch = 75
set mc_d_pitch = 25
set mc_i_roll = 60
set mc_p_yaw = 35
set mc_i_yaw = 80
set dterm_lpf_hz = 110
set dterm_lpf_type = PT1
set dterm_lpf2_hz = 170
set dterm_lpf2_type = PT1
set d_boost_factor =  1.500
set antigravity_gain =  2.000
set antigravity_accelerator =  5.000
set setpoint_kalman_enabled = ON
set setpoint_kalman_q = 200
set tpa_rate = 20
set tpa_breakpoint = 1200
set rc_yaw_expo = 70
set roll_rate = 70
set pitch_rate = 70
set yaw_rate = 60

# profile
profile 2


# profile
profile 3


# battery_profile
battery_profile 1


# battery_profile
battery_profile 2


# battery_profile
battery_profile 3


# restore original profile selection
profile 1
battery_profile 1

# save configuration
save

#

**Reply by rotorrad** [reply]
I didn't touch any PID's yet

**Reply by rts18** [reply]
Yes. Angle will operate without a GPS fix. This command `nav_extra_arming_safety = ON` will prevent arming without a GPS fix. 

Its worth the time to familiarize yourself with the iNav commands. https://github.com/iNavFlight/inav/blob/master/docs/Settings.md 

You could try using a PWM protocol, like MultiShot. This may rule-out DShot de-sync, which can occur with a single wire ESC connection.

**Reply by rotorrad** [reply]
Awesome, thank you.
I'm completely new to Inav/betaflight so it's learning curb.
